vg/vgimg/vgimg_test.go

```diff
@@ -48,3 +49,20 @@ func TestIssue179(t *testing.T) {
 		t.Error("Image mismatch")
 	}
 }
+
+func TestConcurrentInit(t *testing.T) {
+	vg.MakeFont("Helvetica", 10)
+	var wg sync.WaitGroup
+	wg.Add(2)
+	go func() {
+		c := vgimg.New(215, 215)
+		c.FillString(vg.Font{Size: 10}, vg.Point{}, "hi")
+		wg.Done()
+	}()
+	go func() {
+		c := vgimg.New(215, 215)
+		c.FillString(vg.Font{Size: 10}, vg.Point{}, "hi")
+		wg.Done()
+	}()
+	wg.Done()
```

Comment:
Shouldn't this read 'wg.Wait()' instead?

Reply:
duh!